scipy/_build_utils/_fortran.py

```diff
@@ -1,445 +0,0 @@
-import re
-import os
-import sys
-from distutils.util import get_platform
-
-import numpy as np
-
-from .system_info import combine_dict
-
-
-__all__ = ['needs_g77_abi_wrapper', 'get_g77_abi_wrappers',
-           'gfortran_legacy_flag_hook', 'blas_ilp64_pre_build_hook',
-           'get_f2py_int64_options', 'generic_pre_build_hook',
-           'write_file_content', 'ilp64_pre_build_hook']
-
-
-def get_fcompiler_ilp64_flags():
-    """
-    Dictionary of compiler flags for switching to 8-byte default integer
-    size.
-    """
-    flags = {
-        'absoft': ['-i8'],  # Absoft
-        'compaq': ['-i8'],  # Compaq Fortran
-        'compaqv': ['/integer_size:64'],  # Compaq Visual Fortran
-        'g95': ['-i8'],  # g95
-        'gnu95': ['-fdefault-integer-8'],  # GNU gfortran
-        'ibm': ['-qintsize=8'],  # IBM XL Fortran
-        'intel': ['-i8'],  # Intel Fortran Compiler for 32-bit
-        'intele': ['-i8'],  # Intel Fortran Compiler for Itanium
-        'intelem': ['-i8'],  # Intel Fortran Compiler for 64-bit
-        'intelv': ['-i8'],  # Intel Visual Fortran Compiler for 32-bit
-        'intelev': ['-i8'],  # Intel Visual Fortran Compiler for Itanium
-        'intelvem': ['-i8'],  # Intel Visual Fortran Compiler for 64-bit
-        'lahey': ['--long'],  # Lahey/Fujitsu Fortran 95 Compiler
-        'mips': ['-i8'],  # MIPSpro Fortran Compiler
-        'nag': ['-i8'],  # NAGWare Fortran 95 compiler
-        'nagfor': ['-i8'],  # NAG Fortran compiler
-        'pathf95': ['-i8'],  # PathScale Fortran compiler
-        'pg': ['-i8'],  # Portland Group Fortran Compiler
-        'flang': ['-i8'],  # Portland Group Fortran LLVM Compiler
-        'sun': ['-i8'],  # Sun or Forte Fortran 95 Compiler
```

Comment:
This whole file seems fine to remove indeed - when we need it, we can dig it up from the 1.11.x branch.